### DISCUSSION TITLE
An issue accured when I tried to start minecraft

### DISCUSSION THREAD

**Discussion by CuriousBoy0** [sandboxie-plus/Sandboxie#3722] [Q&A Feedback]
Hello,

I'm using Sandboxie-Plus version `v1.12.9`.
I've set up a sandbox at `C:\Sandbox\Common` and named it "Common".
Here's my configuration:

> Enabled=y,Tamim,Alwal
> BlockNetworkFiles=y
> RecoverFolder=%{374DE290-123F-4565-9164-39C4925E467B}%
> RecoverFolder=%Personal%
> RecoverFolder=%Desktop%
> BorderColor=#02f6f6,ttl
> Template=OpenBluetooth
> Template=SkipHook
> Template=FileCopy
> Template=qWave
> Template=BlockPorts
> Template=LingerPrograms
> Template=AutoRecoverIgnore
> ConfigLevel=10
> FileRootPath=C:\Sandbox\Common
> UseFileDeleteV2=y
> UseRegDeleteV2=y
> SeparateUserFolders=n
> AutoRecover=y
> WriteFilePath=C:\Users
> ClosedFilePath=C:\DRIVERS
> ClosedFilePath=C:\RetroArch-Win64
> ClosedFilePath=C:\RyzenPPKG Driver
> ClosedFilePath=C:\AMD
> ClosedFilePath=C:\Autodesk
> ClosedFilePath=C:\HP
> OpenFilePath=C:\Temp
> DllInjection=Block
> ClosePrintSpooler=y
> OpenClipboard=n
> DropAdminRights=y
> FakeAdminRights=y
> ForceFolder=C:\Sandbox\Common

Alwal is the admin, and Tamim is the user.
I utilize ASTER ibik, a software for enabling and managing multiple user sessions simultaneously.
Minecraft is installed at:
`C:\Sandbox\Common\Drive\C\Users\Alwal\AppData\Roaming\.tlauncher`
It works fine. However, when I attempt to play it from my brother's account while I'm using it, Sandboxie returns these issues on my brother's monitor:
![IMG_20240320_055505](https://github.com/sandboxie-plus/Sandboxie/assets/163960511/213cf940-a28d-4aa1-b694-fe6661c55a5e)
Interestingly, after closing Minecraft and rerunning it on my brother's monitor , it works. Now, it's installed at:
`C:\Sandbox\Common\Drive\C\Users\Tamim\AppData\Roaming\.tluancher`
To avoid data replication and share mods, versions, saved worlds, and shaderpacks, I use the path:
`C:\Sandbox\Common\Drive\C\ProgramData\.minecraft`
While Minecraft is running on my brother's screen, I attempted to run it on mine but encountered the same issues again.
I want to play Minecraft with my brother simultaneously on a LAN server without duplicating the entire drive or Minecraft folders, and also play multiplayer with him on an internet server
Creating two different sandboxes would replicate the data. 
Do you have any recommendations, solutions, or techniques that I might not be aware of?

Sorry, If my English is bad or not clear, I have used chatGPT to Write it
Here is my last note It is not only minecraft issue 
I can't run firefox twice on the same sandbox even when the user is different

**Comment by offhub** [comment]
It will not work with a single sandbox. Multiple users cannot use the same registry hive at the same time.

1. Create a second box with the same settings.
2. Run unsandboxed CMD as admin.
3. Link the sandbox locations through the host using the following command. This will link `Common\...\.minecraft` to the `SecondBox\...\.minecraft` folder. (See `https://www.google.com/search?q=symlink+windows`)
```
mklink /D "C:\Sandbox\SecondBox\Drive\C\ProgramData\.minecraft" "C:\Sandbox\Common\Drive\C\ProgramData\.minecraft"
```
4. Add the following rule to the second sandbox.
```
OpenPipePath=C:\Sandbox\Common\Drive\C\ProgramData\.minecraft\
```
5. (Optional) If necessary, you can link other locations in the same way.


> [!NOTE]
> I haven't tested it with Minecraft.

**Reply by CuriousBoy0** [reply]
Thx for replying :)
